source/_posts/2016-03-24-the-data-center-operating-system-installer-part-1.markdown

```diff
@@ -5,21 +5,21 @@ date: 2016-03-24 08:54:43 -0700
 comments: true
 categories: 
 ---
-The Data Center Operating System (DCOS) is a distributed, highly available task scheduler built by [Mesosphere](http://mesosphere.io). It uses a number of open and close source projects to make running and administering Apache Mesos as seamless and simple as possible. The DCOS runs at scale (we have customers running production deployments of 50,000 nodes), across thousands of machines. This post covers challenges, design and an overview of the final GUI installer we built to install an operating system for the data center.  
+The [Mesosphere](http://mesosphere.io) Data Center Operating System (DCOS) is a distributed, highly available task scheduler. It uses a number of open and closed source projects to make running and administering Apache Mesos as seamless and simple as possible. DCOS runs at scale (we have customers running production deployments of 50,000 nodes) across thousands of machines. This post covers challenges, design, and an overview of the final GUI installer we built to install an operating system for the data center. <!-- not sure about that last sentence --> 
 <!--more-->
```

Comment:
This post covers some of the challenges we encountered creating the GUI installer, as well an overview of the final design.